server/commands.lua

```diff
@@ -113,42 +69,66 @@ QBCore.Commands.Add('tp', Lang:t("command.tp.help"), { { name = Lang:t("command.
             TriggerClientEvent('QBCore:Notify', source, Lang:t('error.missing_args'), 'error')
         end
     end
-end, 'admin')
+end)
+
 
-QBCore.Commands.Add('tpm', Lang:t("command.tpm.help"), {}, false, function(source)
+lib.addCommand('tpm', {
+    help = Lang:t("command.tpm.help"),
+    restricted = "qbox.admin"
+}, function(source, _)
     TriggerClientEvent('QBCore:Command:GoToMarker', source)
-end, 'admin')
+end)
 
-QBCore.Commands.Add('togglepvp', Lang:t("command.togglepvp.help"), {}, false, function()
+lib.addCommand('togglepvp', {
+    help = Lang:t("command.togglepvp.help"),
+    restricted = "qbox.god"
+}, function(_, _)
     QBConfig.Server.PVP = not QBConfig.Server.PVP
     TriggerClientEvent('QBCore:Client:PvpHasToggled', -1, QBConfig.Server.PVP)
-end, 'admin')
+end)
 
 -- Permissions
 
-QBCore.Commands.Add('addpermission', Lang:t("command.addpermission.help"), { { name = Lang:t("command.addpermission.params.id.name"), help = Lang:t("command.addpermission.params.id.help") }, { name = Lang:t("command.addpermission.params.permission.name"), help = Lang:t("command.addpermission.params.permission.help") } }, true, function(source, args)
-    local Player = QBCore.Functions.GetPlayer(tonumber(args[1]))
-    local permission = tostring(args[2])
+lib.addCommand('addpermission', {
+    help = Lang:t("command.addpermission.help"),
+    params = {
+        {name = Lang:t("command.addpermission.params.id.name"), help = Lang:t("command.addpermission.params.id.help")},
+        {name = Lang:t("command.addpermission.params.permission.name"), help = Lang:t("command.addpermission.params.permission.help")}
+    },
+    restricted = "qbox.god"
+}, function(source, args)
+    local Player = QBCore.Functions.GetPlayer(tonumber(args[Lang:t("command.addpermission.params.id.name")]))
```

Comment:
nit; local variable should be camelCase

Reply:
so i guess u mean permission should be Permission?

Reply:
local Player -> local player

Reply:
Oh yeah makes sense, thought it was using Capital letters for each words